src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -1063,6 +1063,131 @@ void CheckChain()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(~TestPlatforms.Linux)]
+        public static void BuildChainForFraudulentCertificate()
+        {
+            byte[] certBytes = Convert.FromBase64String(@"
```

Comment:
Can you include some of the same comments from your PR as comments in the source? It's otherwise not clear to a reader what these blobs are or where they came from.